src/backend/gl/src/queue.rs

```diff
@@ -8,6 +8,7 @@ use crate::{
     command as com, device, info::LegacyFeatures, native, state, Backend, GlContext, Share, Starc,
     Surface, Swapchain,
 };
+use parking_lot::Mutex;
```

Comment:
super nit: the use statements are currently grouped as: standard library, then external dependencies, then local dependencies. So `parking_lot` should go near `glow` and `smallvec` ones.

Reply:
Ah yes, I like doing that with my use statements too.